src/Robo/Tasks/DrushTask.php

```diff
@@ -245,4 +249,84 @@ protected function mixedToBool($mixedVar) {
     return $boolVar;
   }
 
+  /**
+   * Set the options to be used for each drush command in the stack.
+   */
+  protected function setGlobalOptions() {
+    if (!empty($this->uri)) {
+      $this->globalOption("uri={$this->uri}");
+    }
+
+    if (isset($this->assume) && is_bool($this->assume)) {
+      $assumption = $this->assume ? 'yes' : 'no';
+      $this->globalOption($assumption);
+    }
+
+    if ($this->verbosityThreshold() >= VerbosityThresholdInterface::VERBOSITY_VERBOSE) {
+      $this->verbose(TRUE);
+    }
+
+    if ($this->verbose) {
+      $this->globalOption('verbose');
+    }
+
+    if ($this->include) {
+      $this->globalOption("include={$this->include}");
+    }
+  }
+
+  /**
+   * Set an option to be used for each drush command in the stack.
+   *
+   * @param string $option
+   * @param string $value
+   *
+   * @return $this
+   */
+  public function globalOption($option, $value = null) {
+    if ($option !== null and strpos($option, '-') !== 0) {
```

Comment:
I don't totally understand why it's necessary to override parent::option and also have a separate globalOption method. Can you explain and perhaps add a comment to the code with the explanation?